Formula/gnupg.rb

```diff
@@ -49,11 +50,22 @@ def install
     system "make"
     system "make", "check"
     system "make", "install"
+
+    # Configure scdaemon as recommended by upstream developers
+    # https://dev.gnupg.org/T5415#145864
+    on_macos do
+      # write to buildpath then install to ensure existing files are not clobbered
+      (buildpath/"scdaemon.conf").write <<~EOS
+        disable-ccid
+      EOS
+      pkgetc.install "scdaemon.conf"
+    end
   end
 
   def post_install
     (var/"run").mkpath
     quiet_system "killall", "gpg-agent"
+    quiet_system "killall", "scdaemon"
```

Comment:
For a less "aggressive" reload, there's also `gpgconf --reload scdaemon` (also applicable to `gpg-agent`).

Reply:
🤷 I'll defer that to more experienced gnupg peeps and a separate PR if there's rationale for it